label_studio/users/api.py

```diff
@@ -138,6 +138,14 @@ def retrieve(self, request, *args, **kwargs):
     def partial_update(self, request, *args, **kwargs):
         result = super(UserAPI, self).partial_update(request, *args, **kwargs)
 
+        # throw MethodNotAllowed if read-only fields are attempted to be updated
```

Comment:
What happens if we don't throw this exception? Is writing of the read_only_fields suppressed, but silently (returns with a 2XX ?) Or will it update the read_only_field s?

Reply:
Attempted updating of read_only_fields results in no changes (is suppressed) but doesn't raise any errors by default. From what I read, this is by design. In our case the ticket explicitly asks for an error on this.